src/components/DatePicker.vue

```diff
@@ -132,8 +132,8 @@
     components: { Day },
 
     props: {
-      borderDateColor:{
-        default:null,
+      currentDateStyle:{
+        default:'border: 1px solid #00c690',
         type:String,
```

Comment:
I'd prefer if we use the object syntax for the style here, as it can quickly get cumbersome to deal with long strings of inline styles if the user wants to change multiple CSS properties :)

Ref: https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1

```
type: Object,
default: () => ({ border: "1px solid #00c690" }),
```

Reply:
Yeah you're right, it will be more flexible. I'll work on it now